ansible/bootstrap.yml

```diff
@@ -317,6 +317,8 @@
   become: yes
   tags: k3s
   tasks:
-    - ansible.builtin.include_role:
+    - name: Install k3s
+      when: "'builder' in group_names"
```

Comment:
Could do via [host patterns](https://docs.ansible.com/ansible/latest/inventory_guide/intro_patterns.html) at L316

```ansible
- hosts: k3s:&builder
```

but :shrug: this is clear at least. No change please.

Reply:
Actually changed my mind - I think the agent one should be done via host patterns so I think this should too.